inventory/host_vars/_example.com/secrets.example.yml

```diff
@@ -17,6 +17,20 @@ secret_token: 'xxxxxx'
 # Create a Google Maps API key and enable Javascript and Places libraries
 google_maps_api_key:
 
+# Amazon S3 backups with db2fog
```

Comment:
those get set by default to none
https://github.com/openfoodfoundation/ofn-install/blob/530741c5d82f6c47b0e196666b0b13920310fedf/roles/app/defaults/main.yml

i guess  a comment should clarify that deployments not using S3 should delete those line from secrets or at least comment them out?

Reply:
Our check for including the S3 settings is: `when: s3_backups_bucket != "none"`. Do we need this defaults file or can we just check if `s3_backups_bucket` is set instead?

Reply:
We have variables initialized in `app/main/defaults.yml` and `deploy/defaults/main.yml` with the string "none". Is there a reason it was done like that? We could just have them blank and instead check `when: s3_backups_bucket != ""`, and that way blank variables in secrets files won't have to be commented out. Is that a better solution?

Reply:
I think it makes more sense if people don't have to manually comment things out. I've added a new commit for it.

Reply:
I'd like to investigate why we used `none` in the first place and go one way or another (`none` or `empty`). I agree with your approach @Matt-Yorkley

Reply:
I see `none` used already 2 years ago https://github.com/openfoodfoundation/ofn-install/blame/master/roles/app/tasks/main.yml#L60
I don't recall discussion about `none` or `empty`, I just worry that someone might get impression that shouldn't leave any values empty, but I think we could clarify that one can lave certain values empty or delete them all together (which what I would do in my configuration file)

Reply:
I've updated the wording to: `# Set these if using Amazon S3 backups with db2fog` to clarify that they are optional.

Reply:
I've tested both deployment and provisioning (with and without the keys set in secrets.yml) with this branch on UK staging, and these were no errors. So it looks like we don't need to initialize variables with random strings like "none" or "this_is_an_empty_variable". :+1: